docs/developers/reference/api/linea-gettransactionexclusionstatusv1.mdx

```diff
@@ -0,0 +1,82 @@
+---
+title: linea_getTransactionExclusionStatusV1
+description: Reference content for the linea_getTransactionExclusionStatusV1 method.
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# `linea_getTransactionExclusionStatusV1`
+
+Queries a database maintained by the sequencer to check if a transaction was rejected because it 
+exceeded data line limits that would prevent the prover from generating a proof for the block.
+
+If the transaction was rejected, the API call will be successful and provide the reason. 
+
+Transactions that were not rejected for this reason will return a `Rejected transaction not available` 
+error. 
+
+:::warning
+Transactions older than 24 hours cannot be checked.
```

Comment:
24 hours is TBD, @FlorianHuc @julien-marchand what's you guys suggestion here?

Reply:
I would give more time; between a user reporting a problem and an actual investigation, it can take a while. I'd recommend 7 days.

Reply:
So lets go with 7 days, we will change the application config in devnet and sepolia accordingly

Reply:
cc @zbconsys I will ping u with the deployment PR